test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -2269,6 +2272,7 @@ public void testMetadataColumns_timestampNull() throws Exception {
 
   @Test
   public void testMetadataColumns_clob() throws Exception {
+    assumeFalse("MySQL does not support clobs", is(MYSQL));
```

Comment:
The messages show up in the test output as:
Testcase: testMetadataColumns_array took 0.003 sec
    	SKIPPED: MySQL does not support arrays

I find it useful.